admin/server/projects.go

```diff
@@ -16,13 +16,21 @@ import (
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
-func (s *Server) ListProjects(ctx context.Context, req *adminv1.ListProjectsRequest) (*adminv1.ListProjectsResponse, error) {
+func (s *Server) ListProjectsForOrganization(ctx context.Context, req *adminv1.ListProjectsForOrganizationRequest) (*adminv1.ListProjectsForOrganizationResponse, error) {
 	claims := auth.GetClaims(ctx)
-	if claims.OwnerType() != auth.OwnerTypeUser {
-		return nil, status.Error(codes.Unauthenticated, "not authenticated as a user")
+
+	org, err := s.admin.DB.FindOrganizationByName(ctx, req.OrganizationName)
+	if err != nil {
+		if errors.Is(err, database.ErrNotFound) {
+			return nil, status.Error(codes.InvalidArgument, "org not found")
+		}
+		return nil, status.Error(codes.InvalidArgument, err.Error())
+	}
+	if !claims.CanOrganization(ctx, org.ID, auth.ReadProjects) {
+		return nil, status.Error(codes.PermissionDenied, "does not have permission to read project")
 	}
```

Comment:
This doesn't handle the case where caller is an outside member. Then it should return the projects in the org that the caller is a member of only.

It should also always return public projects here, if there are any (even for anons)